u9/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useContext } from "react";
+import { BaseContext, BaseContextType } from "u9/contexts/base";
+import { colors } from "u9/utils/styles/theme";
+import Logo from "../Logo/Logo";
+import * as Styled from "./Menu.styles";
+
+type MenuProps = {
+  isDark?: boolean;
+};
+const Menu = ({ ...props }: MenuProps) => {
```

Comment:
I can't access the about and services pages when on mobile, is it possible to add the simplest possible menu to give access to such pages?

![localhost_3000_(Pixel 5)](https://user-images.githubusercontent.com/310976/188932799-aba9e513-cbc1-4b2e-945e-4bd1ab6bd57f.png)